spoon/datagrid/datagrid.php

```diff
@@ -1479,7 +1479,7 @@ public function setColumnConfirm($column, $message, $custom = null)
 	 * @param	mixed $columns					The columns wherein the result will appear.
 	 * @param	bool[optional] $overwrite		Should the result overwrite the current value?
 	 */
-	public function setColumnFunction($function, $arguments = null, $columns, $overwrite = false)
+	public function setColumnFunction($function, $arguments = null, $columns = [], $overwrite = false)
```

Comment:
In PHPDoc staat er wel mixed. Dus zou verwachten dat het ook null kan zijn.

Reply:
fixed in 3e50c97a922359dac8cf744cbf09ddd66d5965ea